Cargo.toml

```diff
@@ -64,7 +64,7 @@ slog-async = "2.3"
 slog-json = "2.3"
 slog-term = "2.4"
 tempfile = "3.7"
-tokio = "1.39.0"
+tokio = "1.41.0"
```

Comment:
This version is the first one with this new unstable metric: https://github.com/cloudflare/foundations/pull/93/files#diff-12da74d1ff15173b0d56ab5b85a436f47d810c25998f4f062980bd2da52e28c3R56